### ISSUE TITLE
Should only have a maximum of 5 slots for cards, not 6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jsvazic/double-fanucci-ga#10]
```
The current system allows for up to 6 Fanucci slots, but the game only
supports up to 5.  This should be updated.
```

Original issue reported on code.google.com by `jsva...@gmail.com` on 9 May 2009 at 3:08

**Comment by GoogleCodeExporter** [comment]
```
Also should note that you can only select to have one slot.  The current 
solution
only starts at 2.
```

Original comment by `jsva...@gmail.com` on 9 May 2009 at 3:15
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed for v1.1
```

Original comment by `jsva...@gmail.com` on 9 May 2009 at 5:13
- Changed state: **Fixed**
